### ISSUE TITLE
Change the name of a blend shape

### ISSUE THREAD

**Issue by Fukuro99** [Yellow-Dog-Man/Resonite-Issues#2200]
### Is your feature request related to a problem? Please describe.

There are times when the name of a blend shape is difficult to understand and you want to change it.
In particular, a recent update has made it possible to divide blend shapes, but even if you divide them on the Y or Z axis, the names are still difficult to understand because they are named with L and R, so it would be better to be able to change them later.

### Describe the solution you'd like

As with deleting and splitting, clicking the button on the edge opens a context menu and adds a context button for renaming the blend shape. Then, enter a new name from the keyboard to change the name of the blend shape.

### Describe alternatives you've considered

Edit and load in an external tool such as Blender.

### Requesters

Resonite:fukuro
Discord:fukuro8232

**Comment by Dusty-Sprinkles** [comment]
I think this would be good to have for a number of reasons (adding new sliders to control things inside resonite) but one immediate solution to your problem might be using the method proxy that splits blendshapes and allows you to define the suffix

**Comment by antrobot1234** [comment]
Correct me if I'm wrong, but the i see with this suggestion is that blend shapes names are on the actual model itself. Changing the name of a blend shape would require changing the name stored within the file. even if you did do that, updating the model with a new version would revert it to the original version. The reason why suffixes work is that they don't change the name itself, they just add onto it. There would have to be some sort of mapping that references the original's name and converts it to the new name, rather than overwriting the original's name entirely.

**Comment by Fukuro99** [comment]
Yes, the model shape key names must be rewritten to achieve this functionality.
However, I see no need to retain the changed names when the model is updated.
This is because when the model is replaced, it may include a use to restore the shape key to its original state.

**Comment by Frooxius** [comment]
We might do this one once we rework the inspectors, since it would require more custom UI and mechanisms to implement this.

It's not that complicated issue to actually do the rename itself, but we don't have good UI framework to do this simply yet.
